certbot-nginx/certbot_nginx/display_ops.py

```diff
@@ -0,0 +1,44 @@
+"""Contains UI methods for Nginx operations."""
+import logging
+import os
+
+import zope.component
+
+from certbot import errors
+from certbot import interfaces
+
+import certbot.display.util as display_util
+
+
+logger = logging.getLogger(__name__)
+
+
+def select_vhost_multiple(vhosts):
+    """Select multiple Vhosts to install the certificate for
+    :param vhosts: Available Nginx VirtualHosts
+    :type vhosts: :class:`list` of type `~obj.Vhost`
+    :returns: List of VirtualHosts
+    :rtype: :class:`list`of type `~obj.Vhost`
+    """
+    if not vhosts:
+        return list()
+    tags_list = [vhost.display_repr() for vhost in vhosts]
+    while True:
```

Comment:
https://github.com/certbot/certbot/pull/5608#discussion_r171141004

Reply:
fun fact these links never go to the right spot

Reply:
but I can manually check, so, got it.

Reply:
done